manifests/compose/autoheal/compose.yaml

```diff
@@ -0,0 +1,12 @@
+name: autoheal
+
+services:
+  autoheal:
+    deploy:
+      replicas: 1
+    image: willfarrell/autoheal:latest
+    network_mode: none
+    restart: always
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
@KaiyiLiu1234  , I am not convinced we need autoheal for compose files. 
Too vary of running unverified container and give it docker.sock access.